crates/ethereum/cli/src/app.rs

```diff
@@ -96,7 +98,8 @@ where
                 self.cli.logs.log_file_directory.join(chain_spec.chain().to_string());
         }
 
-        self.init_tracing()?;
+        self.init_tracing(Some(&runner))?;
```

Comment:
do we ever call `self.init_tracing(None)`? if not then maybe we should require the runtime for `init_tracing`, even though it may be unused if grpc is not used